pubspec.yaml

```diff
@@ -20,7 +20,7 @@ dev_dependencies:
 
 environment:
   sdk: ">=2.2.0 <3.0.0"
-  flutter: ">=1.18.0-6.0.pre <2.0.0"
+  flutter: ">=1.17.0 <2.0.0"
```

Comment:
This is not safe. Code was changed in this repo that is not compatible wtih v1.17.0.

If you need to use Flutter v1.17.0, you need to use flutter_svg v0.17.x